packages/rocketchat-sharedsecret/sharedsecret.js

```diff
@@ -0,0 +1,106 @@
+/* globals HandleSlashCommand */
+const SharedSecret = [];
+
+if (Meteor.isServer) {
+	const EncryptMessage = (() => {
```

Comment:
If you want to "make a class", I think that you remove/forgot one `()` here 
```javascript
const Class = (()=>{return function(){}})()
```
nowadays you can just use `class EncryptMessage {...}`

but I think that in all situations could be simple functions.
like:
```javascript
function HandleSlashCommand(command, params, item) {...}
```